apps/dsp-app/src/assets/i18n/de.json

```
@@ -149,30 +149,30 @@
     "properties": "Eigenschaften",
     "comment": "Kommentar",
     "incomingLink": {
-      "tooltip": "Indicates that this resource is referred to by another resource",
-      "label": "has incoming link"
+    "tooltip": "Zeigt an, dass auf diese Ressource von einer anderen Ressource verwiesen wird",
+    "label": "hat eingehenden Link"
     },
     "standoffLink": {
-      "tooltip": "Represent a link in standoff markup from one resource to another",
-      "label": "has Standoff link"
+    "tooltip": "Stellt einen Link in Standoff-Markup von einer Ressource zu einer anderen dar",
+    "label": "hat Standoff-Link"
     },
     "footnotes": "Fussnoten",
     "permissions": {
       "groups": {
         "label": "Berechtigungen",
-        "projectAdmin": "Project Admin",
-        "creator": "Creator",
-        "projectMember": "Project Member",
-        "knownUser": "Known User",
-        "unknownUser": "Unknown User",
-        "usersPermissions": "Your permissions"
+        "projectAdmin": "Projektadministrator",
+        "creator": "Ersteller",
+        "projectMember": "Projektmitglied",
+        "knownUser": "Bekannter Benutzer",
+        "unknownUser": "Unbekannter Benutzer",
+        "usersPermissions": "Ihre Berechtigungen"
       },
       "interactions": {
-        "restrictedView": "Restricted view permission (RV)",
-        "view": "View permission (V)",
-        "modify": "Modify permission (M)",
-        "delete": "Delete permission (D)",
-        "changeRights": "Change rights permission (CR)"
+        "restrictedView": "Eingeschränkte Ansichtsberechtigung (RV)",
+        "view": "Ansichtsberechtigung (V)",
+        "modify": "Änderungsberechtigung (M)",
+        "delete": "Löschberechtigung (D)",
+        "changeRights": "Rechteänderungsberechtigung (CR)"
```

```suggestion
        "restrictedView": "Berechtigung mit Einschränkungen anzusehen (RV)",
        "view": "Berechtigung anzusehen (V)",
        "modify": "Berechtigung zu bearbeiten (M)",
        "delete": "Berechtigung zu löschen (D)",
        "changeRights": "Berechtigung Rechte zu vergeben (CR)"
```

@danielasubotic what do you think of these suggestions?